### DISCUSSION TITLE
Implement CogVideo

### DISCUSSION THREAD

**Discussion by sgtpepper67** [lllyasviel/stable-diffusion-webui-forge#1914] [Ideas]
An open source video generator that is currently better than any other locally run video generator, and it has relatively low weights so GPUs with lower VRAM could run it.
https://github.com/THUDM/CogVideo

**Comment by maraan666** [comment]
This would be wonderful.

**Comment by pflky** [comment]
It would just be added to the "spaces" section, which would be virtually no different than using it through Pinokio, so you don't need forge to add it to use it right now with a 1 click install. 

I would only want this in forge of it was improved upon and integrated into the main UI, not through "spaces". I'd want it to be able to use the NF4/GGUF models, not the main full sized flux model that is baked into it. That's really what makes it pretty horrible right now. It takes a good 10-15 minutes to generate.

**Reply by sgtpepper67** [reply]
I meant as an additional tab to the right of img2img, basically like other video generators used to be.

**Reply by maraan666** [reply]
Exactly. As a replacement (or alternative) for SVD.
